### PR TITLE
chore(libzkp): upgrade to v0.11.1

### PR DESCRIPTION
**Pull request by lispc**
+ halo2-gpu v0.8.0 

Highlight:

assets changed:

layer*.config

params:
chunk prover now needs params20 params24 params25
batch prover now needs params21 params26

### CODE REVIEW

File: prover/core/prover.go [hunk 1]
```diff
@@ -3,8 +3,8 @@
 package core
 
 /*
-#cgo LDFLAGS: -lzkp -lm -ldl -lzktrie -L${SRCDIR}/lib/ -Wl,-rpath=${SRCDIR}/lib
-#cgo gpu LDFLAGS: -lzkp -lm -ldl -lgmp -lstdc++ -lprocps -lzktrie -L/usr/local/cuda/lib64/ -lcudart -L${SRCDIR}/lib/ -Wl,-rpath=${SRCDIR}/lib
+#cgo LDFLAGS: -lzkp -lm -ldl -L${SRCDIR}/lib/ -Wl,-rpath,${SRCDIR}/lib
+#cgo gpu LDFLAGS: -lzkp -lm -ldl -lgmp -lstdc++ -lprocps -L/usr/local/cuda/lib64/ -lcudart -L${SRCDIR}/lib/ -Wl,-rpath,${SRCDIR}/lib
```
Comment (lispc) [inline prover/core/prover.go]: i am not sure why this change happend... due to merging??... please review this carefully..

Comment (colinlyguo) [reply prover/core/prover.go]: got it. in original implementation it's `-rpath=${SRCDIR}/lib`, but both works. will revert this change in this PR.

Comment (colinlyguo) [inline prover/core/prover.go]:
```suggestion
#cgo gpu LDFLAGS: -lzkp -lm -ldl -lgmp -lstdc++ -lprocps -L/usr/local/cuda/lib64/ -lcudart -L${SRCDIR}/lib/ -Wl,-rpath=${SRCDIR}/lib
```

**Review by colinlyguo — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -4,6 +4,8 @@ assets/seed
 
 # Built binaries
 build/bin
+verifier.test
```
Comment (amoylan2) [inline .gitignore:7]: where is this binary from?

Comment (colinlyguo) [reply .gitignore:7]: I think it's generated by `common/libzkp/e2e-test.sh`.

Comment (amoylan2) [reply .gitignore:7]: is this file used? since core/prover_test.go is invalid now

Comment (amoylan2) [reply .gitignore:7]: Resolve this and talk in another thread

File: common/libzkp/e2e-test.sh [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+REPO=$(realpath ../..)
+
+function build_test_bins() {
+    cd impl
+    cargo build --release
+    ln -f -s $(realpath target/release/libzkp.so) $REPO/prover/core/lib
+    ln -f -s $(realpath target/release/libzkp.so) $REPO/coordinator/internal/logic/verifier/lib
+    cd $REPO/prover
+    go test -tags="gpu ffi" -timeout 0 -c core/prover_test.go
```
Comment (amoylan2) [inline common/libzkp/e2e-test.sh:21]: is this file used? since core/prover_test.go is invalid now

Comment (colinlyguo) [reply common/libzkp/e2e-test.sh:21]: makes sense, emmm, but we have not added the rust version of this e2e-test script. what about removing them in this PR, and add one in next PR?

Comment (amoylan2) [reply common/libzkp/e2e-test.sh:21]: It's not the main part, can be leaved to next patch

**Review by amoylan2 — APPROVED** [review]

File: common/libzkp/e2e-test.sh (hunk 3 above)
Comment (lispc) [reply common/libzkp/e2e-test.sh:21]: i think the script is not useful now.. can be deleted later
